### ISSUE TITLE
[Bug] Error loading the hosted page

### ISSUE THREAD

**Issue by at-the-vr** [pranavvp16/End-to-End-Anime-Recomendation-Model#1]
Hey pranav, I was going across your project and found that your hosted application (on Heroku) is receiving an Application Error (can be because of Heroku removing its Free tier), can you give ir a try ?

**Comment by pranavvp16** [comment]
Yeah cause heroku has stopped it's free services , let me know if there are other alternatives

**Comment by at-the-vr** [comment]
I found [Railway](https://railway.app/) quite useful (its free tier has a request per month structure)
